src/attacks/square/square_attack.jl

```diff
@@ -11,20 +11,20 @@ function SquareAttack(
     model,
     x,
     y;
-    iterations = 10,
-    ϵ = 0.3,
-    p_init = 0.8,
-    min_label = 0,
-    max_label = 9,
-    verbose = false,
-    clamp_range = (0, 1),
-    loss = nothing,
+    iterations=10,
+    ϵ=0.3,
+    p_init=0.8,
+    min_label=0,
+    max_label=9,
+    verbose=false,
+    clamp_range=(0, 1),
+    loss=nothing,
 )
     Random.seed!(0)
     n_features = length(x)
 
     # Initialization (stripes of +/-ϵ)
-    init_δ = rand(w, 1, c) .|> x -> x < 0.5 ? -ϵ : ϵ
+    init_δ = (x -> x < 0.5 ? -ϵ : ϵ).(rand(w, 1, c))
```

Comment:
Things like this do not change the functionality of the step, right? (not just this but in general towards all the tweaks within the attack algorithms)